### PR TITLE
package.json file was added

### PR DESCRIPTION
**Pull request by Mabonye**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,6 +23,51 @@ app.use(cors());
...
+      query = knex.select('*').from('meal').where('when_date', '>', now);
+      break;
+    case "past-meals":
+      query = knex.select('*').from('meal').where('when_date', '<', now);
+      break;
+    case "all-meals":
+      query = knex.select('*').from('meal').orderBy('id');
+      break;
+    case "first-meal":
+      query = knex.select('*').from('meal').where("id", 1).first();
```
Comment (veronicaburduzhan) [inline src/backend/app.js]: I have a few questions regarding this query:
1) Are you sure you have id=1? What's going to happen if you delete a meal with id=1 before executing this query? 
2) How many meals you're expecting to have with id=1 if you've added first() in the end of the query? Id should be unique, that's why when you specify it in query you always should get a single item.

**Review by veronicaburduzhan — APPROVED** [review]
I like how you decided to approach this task using switch🤩 Almost everything looks good (check my comment).

File: src/backend/app.js (hunk 1 above)
Comment (Mabonye) [reply src/backend/app.js]: > I have a few questions regarding this query:
> 
> 1. Are you sure you have id=1? What's going to happen if you delete a meal with id=1 before executing this query?
> 2. How many meals you're expecting to have with id=1 if you've added first() in the end of the query? Id should be unique, that's why when you specify it in query you always should get a single item.

Hello Veronica, thanks for the review, I will go through the comments and modify the queries:)

**Comment by Mabonye** [comment]
> I like how you decided to approach this task using switch🤩 Almost everything looks good (check my comment).

Hello Veronica,
Thanks for you time and comment:)

**Pull request merged**